### PR TITLE
Adds SOP books

### PR DESCRIPTION
**Pull request by pinatacolada**
Adds SOP books to the corresponding departments and department head's
offices. Rep and IAA have a brand spanking new shelf with all the SOP
books.

Also right now it's using generic book sprites because I couldn't sprite
even if my life depended on it, maybe someone could sprite something
nice for them?

Also fixed a random < that was just kinda there out in nowhere in an
atmos book

Also, just for reference on the shelves
NT Rep

IAA

### CODE REVIEW

File: code/game/objects/items/weapons/manuals.dm [hunk 1]
```diff
@@ -1183,3 +1183,891 @@
...
+				<ol>
+					<li>The Chief Medical Officer is permitted to carry a regular Defibrillator or a Compact Defibrillator on their person at all times;</li>
+                    <li>The Chief Medical Officer is permitted to carry a telescopic baton. In case Genetic Powers need to be forcefully removed, they are cleared to carry a Syringe Gun;</li>
+                    <li>The Chief Medical Officer is not permitted to allow the creation of poisonous or explosive mixtures in Chemistry without express consent from the Captain or, failing that, the presence of a clear and urgent danger to the integrity of the station, except of course in situations where Chemical Implants are required;</li>
+                    <li>The Chief Medical Officer is not permitted to allow the release of any virus without a full list of its symptoms, as well as the creation of a vial of antibodies, to be kept in a secure location. The virus may not have any harmful symptoms whatsoever, though neutral/harmless symptoms are permitted;</li>
+                    <li>The Chief Medical Officer must make sure that any cloneable corpses are, in fact, clone.</li>
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/manuals.dm]: The Chief Medical Officer must make sure that any cloneable corpses are, in fact, **cloned**

File: code/game/objects/items/weapons/manuals.dm [hunk 2]
```diff
@@ -1183,3 +1183,891 @@
...
+                    <li>Surgeons are expected to fulfill the duties of regular Medical Doctors if there are no active Surgical Procedures undergoing; </li>
+                    <li>Medical Doctors must ensure there is at least one (1) Defibrillator available for use, at all times, next to or near the Cryotubes;</li>
+                    <li>Medical Doctors must maintain the entirety of Medbay in an hygienic state. This includes, but is not limited to, cleaning organic residue, fluids and corpses;</li>
+                    <li>Medical Doctors must place all corpses inside body bags. If there is an assigned Coroner, the Morgue Trays must be correctly tagged;</li>
+                    <li>Medical Doctors must, together with Geneticists and Chemists, make sure that Cloning is stocked with Biomass. In addition, Medical Doctors must make sure that the Morgue does not contain cloneable corpses;</li>
+                    <li>Medical Doctors must certify that all cloned personnel are put in the Cryotubes after Cloning, and receive either a dose of Mutadone or a Clean SE Injector, in addition to Mannitol. An exception is made if the Cloning Pod was fully upgraded by Science;</li>
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/manuals.dm]: Medical Doctors must _**ensure**_

File: code/game/objects/items/weapons/manuals.dm [hunk 3]
```diff
@@ -1183,3 +1183,891 @@
...
+                </ol>
+                <br><br>
+
+                <h1><a name="chap"><B>Chaplain</B></a></h1><BR>
+				<ol>
+					<li>The Chaplain is not permitted to execute Bible Healing without consent, unless the person in question is in Critical Condition and there are no doctors, as doing so incurs the risk of causing brain damage;</li>
+                    <li>The Chaplain may not draw the Null Rod or Holy Sword on any personnel. Using these items on any personnel is grounds to have these items confiscated, unless there is a clear and present danger to their life;</li>
+                    <li>The Chaplain may not actively discriminate against any personnel on the grounds that it is a religious tenet of their particular faith;</li>
+                    <li>The Chaplain may not perform funerals for any personnel that have since been cloned;</li>
+                    <li>The Chaplain may, however, freely conduct funerals for non-cloneable personnel. All funerals must be concluded with the use of the Mass Driver or Crematorium.</li>
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/manuals.dm]: non-cloneable to unclonable, just for consistency

File: code/game/objects/items/weapons/manuals.dm [hunk 4]
```diff
@@ -1183,3 +1183,891 @@
...
+                <br><br>
+
+                <h1><a name="cor"><B>Coroner Procedures</B></a></h1><BR>
+                <ol>
+					<li>For the sake of hygiene, the Coroner should wear a Sterile Mask when handling corpses;</b></li>
+                    <li>The Coroner must inject/apply Formaldehyde to all corpses, and place them in body bags;</li>
+                    <li>The Coroner must perform a full autopsy on all corpses, and keep a record of it, in written format. If foul play is suspected, Security must be contacted;</li>
+                    <li>The Coroner must correctly tag the Morgue Trays in order to identify the corpse within, as well as Cause of Death;</li>
+                    <li>The Coroner must ensure Security-based DNR Notices (such as executed personnel, for instance) are respected;</li>
+                    <li>The Coroner must ensure that every ID from unclonable bodies is delivered to either the relevant Head of Staff, or the Head of Personnel. This applies to any Medbay personnel placing a body in the Morgue</li>
```
Comment (KasparoVy) [inline code/game/objects/items/weapons/manuals.dm]: cloneable

File: code/game/objects/items/weapons/manuals.dm (hunk 2 above)
Comment (pinatacolada) [inline code/game/objects/items/weapons/manuals.dm]: you mean make ensure bold?

File: code/game/objects/items/weapons/manuals.dm (hunk 3 above)
Comment (pinatacolada) [inline code/game/objects/items/weapons/manuals.dm]: even if it's a minor thing, not really comfortable make that kind of changes by my own, so, @TullyBurnalot

File: code/game/objects/items/weapons/manuals.dm (hunk 4 above)
Comment (pinatacolada) [inline code/game/objects/items/weapons/manuals.dm]: @TullyBurnalot

File: code/game/objects/items/weapons/manuals.dm (hunk 2 above)
Comment (Tastyfish) [inline code/game/objects/items/weapons/manuals.dm]: He means "ensure" instead of "certify"

**Comment by Tastyfish** [comment]
You have a step_x in cyberiad.dmm.

**Comment by Regen1** [comment]
:+1: Now I can literally beat someone to death with the SOP when they break it

File: code/game/objects/items/weapons/manuals.dm (hunk 2 above)
Comment (pinatacolada) [inline code/game/objects/items/weapons/manuals.dm]: @TullyBurnalot

**Comment by Fox-McCloud** [comment]
Maybe I'll finally have fuel to keep warm now.

**Comment by Crazylemon64** [comment]
@Fox-McCloud That would imply there are no other books without much use to most players.

**Comment by ProperPants** [comment]
mfw books are considered weapons :confused:
The more books have actual text, the better :+1:

**Comment by KasparoVy** [comment]
I'd do a full on spellcheck of this but I'm currently pulling consecutive all nighters for end of term work lmao

if this' still waiting a merge in a day or so I'll go deep and check but otherwise its probably just fine anyway, just me nitpicking/grammar naziing

**Pull request merged**